instrumentation/vertx/vertx-redis-client-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/vertx/v4_0/redis/VertxRedisClientAttributesGetter.java

```diff
@@ -19,36 +20,39 @@ public enum VertxRedisClientAttributesGetter
       RedisCommandSanitizer.create(AgentCommonConfig.get().isStatementSanitizationEnabled());
 
   @Override
-  public String getSystem(VertxRedisClientRequest request) {
-    return DbIncubatingAttributes.DbSystemValues.REDIS;
+  public String getDbSystem(VertxRedisClientRequest vertxRedisClientRequest) {
+    return REDIS;
   }
 
+  @Deprecated
   @Override
   @Nullable
   public String getUser(VertxRedisClientRequest request) {
     return request.getUser();
   }
 
-  @Override
   @Nullable
-  public String getName(VertxRedisClientRequest request) {
+  @Override
```

Comment:
can you move the `@Override` back up above the `@Nullable` just to keep diffs down? thanks